docker/Dockerfile.build.aarch64-linux

```diff
@@ -175,6 +175,7 @@ CMD cmake \
   -DBUILD_NVJPEG=OFF \
   -DBUILD_NVOF=OFF \
   -DBUILD_NVDEC=OFF \
+  -DBUILD_LIBSND=OFF \
```

Comment:
Any reason for it to be off here and on in QNX?

Reply:
Turned on